<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{margin:0in;
	margin-bottom:.0001pt;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";}
.MsoChpDefault
	{font-family:"Calibri","sans-serif";}
.MsoPapDefault
	{margin-bottom:10.0pt;
	line-height:115%;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;}
div.WordSection1
	{page:WordSection1;}
-->
</style>

<div class=WordSection1>

<p class=MsoNoSpacingCxSpMiddle style='text-align:justify'><span lang=EN-GB
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Table showing
the number of sequences identified for each samples and the number of sequences
in which no MID tag could be matched. In addition, multiple files can be
downloaded for each sample.</span></p>

<p class=MsoNoSpacingCxSpMiddle style='text-align:justify'><u><span lang=EN-GB
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>FASTQ:</span></u><span
lang=EN-GB style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
Downloads a FASTQ file with all the sequences that are linked to the sample.
Sequences have not been trimmed. </span></p>

<p class=MsoNoSpacingCxSpMiddle style='text-align:justify'><u><span lang=EN-GB
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>FASTA:</span></u><span
lang=EN-GB style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
Downloads a FASTA file with all the sequences that are linked to the sample.
Sequences have not been trimmed. </span></p>

<p class=MsoNoSpacingCxSpMiddle style='text-align:justify'><u><span lang=EN-GB
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>Trimmed FASTA:</span></u><span
lang=EN-GB style='font-size:12.0pt;font-family:"Times New Roman","serif"'>
Downloads a trimmed FASTA file with all the sequences that are linked to the
sample. Sequences are trimmed according to the settings entered at the barcodes
section of the main page of the demultiplex tool. </span></p>

<p class=MsoNoSpacingCxSpLast style='text-align:justify'><u><span lang=EN-GB
style='font-size:12.0pt;font-family:"Times New Roman","serif"'>FASTQC:</span></u><span
lang=EN-GB style='font-size:12.0pt;font-family:"Times New Roman","serif"'> The
sequences of each individual sample automatically is analysed using FASTQC.
Clicking “Report” opens a new page containing all quality control results of
the analysis of FASTQC. </span><span style='font-size:12.0pt;font-family:"Times New Roman","serif";
color:black'>(</span><span lang=EN-GB><a
href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank"><span
lang=EN-US style='font-size:12.0pt;font-family:"Times New Roman","serif";
color:#303030;text-decoration:none'>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></a></span><span
style='font-size:12.0pt;font-family:"Times New Roman","serif";color:black'>)</span></p>

<p class=MsoNormal>&nbsp;</p>

</div>
